test/operators.jl

```diff
@@ -38,6 +38,16 @@
             @test ne(z) == 1
         end
 
+        @testset "symmetric difference for Digraph" begin
+            g1 = path_digraph(2)
+            g2 = path_digraph(4)
+
+            y = symmetric_difference2(g1, g2)
+            @test y == symmetric_difference2(g2, g1)
```

Comment:
where is this function defined? (Is it left over from testing?)

Reply:
it is a mistake.

Reply:
OK, Cool. Just FYI, things are slowing down on the 2.0.0 changes so I'm ready to review the backlog. Thanks for being so patient.